packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
```

Comment:
The `make installs` calls `ocamlfind install`, it seems like the most convenient way to install the library in order to get ocamlfind integration and correct permissions on `bisect_ppx.byte` to be preserved. I moved the command into the `build` section because it is not execute in the install section because of the conflict (?) with `bisect.install` in the files directory (which is used to copy over the `bisect-ppx-report` utility.

Point noted about calling `ocamlfind remove bisect_ppx` in the `remove` field.

What would be the _canonical_ way to install this package? The `opam` documentation favors the `bisect.install` file, but I could not find how to get that section to just execute the `ocamlfind install` command and install the reporting tool. I'd be happy to change the install accordingly.